airflow/providers/amazon/aws/hooks/redshift.py

```diff
@@ -16,231 +16,14 @@
 # specific language governing permissions and limitations
 # under the License.
 """Interact with AWS Redshift clusters."""
+import warnings
 
-from typing import Dict, List, Optional, Union
+from airflow.providers.amazon.aws.hooks.redshift_cluster import RedshiftHook  # noqa
+from airflow.providers.amazon.aws.hooks.redshift_sql import RedshiftSQLHook  # noqa
```

Comment:
Tip: Adding `__all__ = ["RedshiftHook", "RedshiftSQLHook"]` also silences Flake8, while making the intention clearer.

Reply:
i like it, thanks